website/project/model.py

```diff
@@ -3815,6 +3841,8 @@ class DraftRegistration(StoredObject):
 
     approval = fields.ForeignField('draftregistrationapproval', default=None)
 
+    status_logs = fields.ForeignField('draftregistrationlog', list=True)
```

Comment:
Rather than DraftReg having a forward-ref to DraftRegLog, why not flip the relationship (Log-> DraftReg)? This would be consistent with NodeLog -> Node.